tests/integration-test/src/tests/utils.rs

```diff
@@ -200,7 +203,7 @@ pub fn add_uprobe(globals: Option<Vec<&str>>) -> Result<String> {
         "Always",
         "uprobe",
         "-f",
-        "malloc",
+        "fwrite",
```

Comment:
Any reason for this change?

Reply:
It seemed to be a better choice because the test itself is using fwrite when saving the log to a file, but it still didn't work upstream.